package.json

```diff
@@ -11,7 +11,6 @@
     "@eva-design/eva": "^2.0.0",
     "@expo/vector-icons": "^10.2.0",
     "@expo/webpack-config": "^0.12.11",
-    "@react-native-community/google-signin": "^4.0.3",
```

Comment:
React native google sign in removed? I remember seeing the other PR having the signin added. Can you confirm whether you're removing or adding oauth for the master branch?

Reply:
This dependency was one that was unused. We're using "expo-google-app-auth"

Reply:
Got it! Thanks for clarifying! Merging to staging now